lib/active_record/virtual_attributes/virtual_total.rb

```diff
@@ -42,7 +43,7 @@ def virtual_total(name, relation, options = {})
       #
       #    def allocated_disk_storage
       #      if disks.loaded?
-      #        disks.blank? ? nil : disks.map(&:size).compact.sum
+      #        disks.map(&:size).compact.sum
```

Comment:
don't think we're ready to make this declaration. yes, soon we will per another PR, but not yet.